<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataLogger</class>
 <widget class="QMainWindow" name="DataLogger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataLogger | STABS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="SideBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="MainBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>18</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/home.svg</normaloff>:/icons/icons/home.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="AlarmButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>18</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/bell.svg</normaloff>:/icons/icons/bell.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="LogoutButton">
    <property name="geometry">
     <rect>
      <x>766</x>
      <y>18</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/log-out.svg</normaloff>:/icons/icons/log-out.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab1_button">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>136</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 1</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="FileView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab4_button">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>304</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 4</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab3_button">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>248</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 3</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Stab2_button">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>192</y>
      <width>103</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stab 2</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/radio.svg</normaloff>:/icons/icons/radio.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="DataPanelButton">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>72</y>
      <width>109</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Panel</string>
    </property>
    <property name="icon">
     <iconset resource="../../DataLogger/resources.qrc">
      <normaloff>:/icons/icons/table.svg</normaloff>:/icons/icons/table.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>File View</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../DataLogger/resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
